serverpool/src/main/java/com/ats/serverpool/network/udp/client/UDPClient.java

```diff
@@ -14,36 +15,36 @@ public UDPClient(Peer peer) {
 
         try {
             this.socket = new DatagramSocket(peer.getPort());
-        } catch(SocketException e) {
+        } catch (SocketException e) {
             System.out.println("Error initalizing DatagramSocket");
             System.out.println(e.getMessage());
         }
     }
-    
-    public String receive() {
-        byte[] buffer = new byte[UDPServer.MAX_PACKET_LEN];
-        DatagramPacket packet = new DatagramPacket(buffer, UDPServer.MAX_PACKET_LEN);
-        
+
+    public Message receive() {
+        byte[] buffer = new byte[UDPClient.MAX_PACKET_LEN];
+        DatagramPacket packet = new DatagramPacket(buffer, UDPClient.MAX_PACKET_LEN);
+
         try {
             this.socket.receive(packet);
-            Message msg = processMessageReceived(new String(receivePacket.getData()));
+            Message msg = proccessMsg(new String(packet.getData()));
 
             return msg;
-        } catch(IOException e) {
+        } catch (IOException e) {
             System.out.println("Error receiving packet from server: ");
             System.out.println(e.getMessage());
         }
 
-        return "";
+        return new Message();
```

Comment:
@aleksajovanovic, you are right!! I udpdated the Message object and here an empty Message object is returned instead.